cea/datamanagement/format_helper/cea4_migrate_db.py

```
@@ -0,0 +1,451 @@
+def delete_files(path, verbose=False):
+    """
+    Delete all files in a directory
+
+    Parameters:
+    - path (str): The path to the directory
+    """
+    try:
+        shutil.rmtree(path)
+        if verbose:
+            print(f"Deleted directory: {path}")
+    except Exception as e:
```

Ruff (0.8.2)

Local variable `e` is assigned to but never used

Remove assignment to unused variable `e`

(F841)